### PR TITLE
feat(remove): Start command

### PR DESCRIPTION
**Pull request by henrikstranneheim**
This PR adds/fixes:

- Removes trailblazer start capability from trailblazer
- Added pre_commit to dev reqs

### How to prepare for test
- [x] ssh to hasta
- [x] activate stage: `us`
- [x] request trailblazer-stage on hasta: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh rm_mip_start`

### How to test
- [x] login to hasta
- [x] do `us`
1. `trailblazer ls`
1. `trailblazer scan`
1. Front-end montioring

### Expected test outcome
1. Check that there are no errors ✅ 
1. Check that there are no errors ✅ 
1. Check that the analyses are monitored ✅ 
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MR
- [x] tests executed by HS
- [x] "Merge and deploy" approved by HS
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

**Implement Plan**
- [ ] Together with https://github.com/Clinical-Genomics/servers/pull/389
- [ ] Together with https://github.com/Clinical-Genomics/cg/pull/708

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
### How to test
- [x] login to hasta
- [x] do `us`
1. `trailblazer ls`
1. `trailblazer scan`
1. Front-end montioring

### Expected test outcome
1. Check that there are no errors ✅ 
Screenshot 2020-09-17 at 07 03 58

1. Check that there are no errors ✅ 
Screenshot 2020-09-17 at 09 58 05

1. Check that the analyses have been monitored ✅ 
Screenshot 2020-09-17 at 09 59 33

File: trailblazer/exc.py [hunk 1]
```diff
@@ -10,9 +7,5 @@ class MissingFileError(TrailblazerError):
     pass
 
 
-class MipStartError(TrailblazerError):
-    pass
-
-
 class ConfigError(TrailblazerError):
     pass
```
Comment (Mropat) [inline trailblazer/exc.py:11]: This we dont need anymore either!

**Review by Mropat — APPROVED** [review]
Looks good!
But, just to be sure, can you also test to install this wth cg master, to make sure no imports are breaking in cg?

File: trailblazer/exc.py (hunk 1 above)
Comment (henrikstranneheim) [reply trailblazer/exc.py]: Done

**Comment by henrikstranneheim** [comment]
> But, just to be sure, can you also test to install this wth cg master, to make sure no imports are breaking in cg?
I am thinking this will be tested and shipped together according to the implementation plan

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
Screenshot 2020-09-22 at 15 20 02

**Pull request merged**